introduction.md

```diff
@@ -0,0 +1,4 @@
+At the moment I'm doing a course mastering github course on www.codeschool.com. The
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.